docs/source/technical_tutorials/dps_tutorial/dps_tutorial_demo.ipynb

```diff
@@ -41,7 +41,7 @@
     "## Before Starting\n",
```

Comment:
<p>Swap the order of recommendations:</p><ol><li>Generate a algorithm yaml using the UI first time (Recommended)</li><li>You can create manually following the template.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>